index.html

```diff
@@ -32,18 +32,22 @@
   </header>
   <main>
     <div class="row roles-icons">
-      <div class="column-half left-1">
-        <img class="role-icon" src="images/duelist.png" alt="Duelist">
+      <!-- <div class="column-half left-1">
+        <img id="duelist" class="role-icon" src="images/duelist.png" alt="Duelist">
       </div>
       <div class="column-half right-1">
-        <img class="role-icon" src="images/initiator.png" alt="Initiator">
+        <img id="initiator" class="role-icon" src="images/initiator.png" alt="Initiator">
       </div>
       <div class="column-half left-2">
-        <img class="role-icon" src="images/controller.png" alt="Controller">
+        <img id="controller" class="role-icon" src="images/controller.png" alt="Controller">
       </div>
       <div class="column-half right-2">
-        <img class="role-icon" src="images/sentinel.png" alt="Sentinel">
-      </div>
+        <img id="sentinel" class="role-icon" src="images/sentinel.png" alt="Sentinel">
+      </div> -->
```

Comment:
Remove all old or commented out code. If you plan on reusing this in a later feature, just save it in a notepad and paste it in when you start the next feature